plugins/woocommerce/tests/e2e-pw/tests/merchant/customer-payment-page.spec.js

```diff
@@ -107,7 +107,7 @@ test.describe(
 			await page.locator( 'label[for=order_status] > a' ).click();
 
 			// pay for the order
-			await page.locator( 'button#place_order' ).click();
+			await page.locator( 'button#place_order' ).dispatchEvent( 'click' );
```

Comment:
Why do we need this?

Reply:
There was some notification bar on WPCOM test sites that I probably removed later, so I will recheck if it is present anymore.